docs/hopsworksai/aws/eks_ecr_integration.md

```diff
@@ -77,9 +77,8 @@ ip-192-168-62-117.us-east-2.compute.internal   Ready    <none>   2m34s   v1.17.9
 
 ## Step 2: Create an instance profile role on AWS
 
-You need to create an instance profile role to allow instances created by Hopsworks.ai to access EKS and ECR.
-To create a role, click on the following [link](https://console.aws.amazon.com/iam/home#/roles$new?step=type&roleType=aws&selectedService=EC2&selectedUseCase=EC2). Alternatively, you can go to the Roles section of the IAM service in AWS management console, click on *Create role*, choose *AWS Service* as the type of trusted entity, and then choose *EC2* from Common use cases. Then, click on *Next: Permissions*, *Next: Tags*, *Next: Review*, and then name your role and click *Create role*.
-Navigate to your newly created role in [AWS management console](https://console.aws.amazon.com/iam/home#/roles) by searching for your role name and click on it. Go to the *Permissions* tab, click on *Add inline policy*, and then go to the *JSON* tab. Paste the following snippet, click on *Review policy*, name it, and click *Create policy*. Finally, copy your Role ARN (you will need it in the next steps).
+You need to add permission to [the instance profile you use for instances deployed by Hopsworks.ai](getting_started.md#step-2-creating-instance-profile) to give them access to EKS and ECR.
+Go to the [*IAM service*](https://console.aws.amazon.com/iam) in the *AWS management console*, click *Roles*, search for your role, and click on it. Click on *Add inline policy*. Go to the *JSON* tab and past the following.
```

Comment:
append the following permissions to the existing JSON permissions.

Reply:
Or do you mean replace? 
replace the existing JSON permissions with the JSON permissions below.